lib/models/movie.js

```diff
@@ -11,5 +12,8 @@ module.exports = Bookshelf.Model.extend({
       release_year: this.get('release_year'),
       object: 'movie'
     };
+  },
+  locations: function () {
```

Comment:
Nit: we usually put `serialize` as the last function defined in a model. It makes it easier to find when we go from model file to model file.